akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FileAndResourceDirectivesSpec.scala

```diff
@@ -91,6 +92,30 @@ class FileAndResourceDirectivesSpec extends RoutingSpec with Inspectors with Ins
         }
       } finally file.delete
     }
+
+    "support precompressed files with registered MediaType" in {
+      val file = File.createTempFile("akkaHttpTest", ".svgz")
+      try {
+        writeAllText("123", file)
+        Get() ~> getFromFile(file) ~> check {
+          mediaType shouldEqual `image/svgz`
```

Comment:
mediaType shouldEqual `image/svg` will also match as they both have the same value string "image/svg".
But I agree that

```scala
mediaType shouldEqual `image/svg`
```

would make the intent a bit clearer.